<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestionProduit</class>
 <widget class="QDialog" name="GestionProduit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Default button style */
#pushButton, #pushButton_4, #pushButton_5, #pushButton_6,
#pushButton_7, #pushButton_8, #pushButton_9, #pushButton_11, 
#pushButton_12, #pushButton_10,#pushButton_2 {
    background-color: #fff;
    width: 10px;
    border: 2px solid #00a388;
    border-radius: 15px;
}



/* Hover button style */
#pushButton:hover, #pushButton_4:hover, #pushButton_5:hover, #pushButton_6:hover,
#pushButton_7:hover, #pushButton_8:hover, #pushButton_9:hover, #pushButton_11:hover, 
#pushButton_12:hover, #pushButton_10:hover,#pushButton_2:hover {
    background-color: #e0f7f4; /* Light green shade */
    border: 2px solid #00a388;
    border-radius: 15px;
}
#Delete{
background-color: rgb(228, 0, 0);
    color: white; /* Texte blanc */
    border: none;
    border-radius: 10px; /* Coins arrondis */
    padding: 8px;}</string>
  </property>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>661</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
    background-color: white;
    width: 10px;
    border: 2px solid #00a388;
    border-radius: 15px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QPushButton" name="pushButton_10">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Add Product</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_9">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>List of Product</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_11">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Statistics</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_12">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Chat Bot</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Notifications</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>231</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
    background-color: white;
    width: 10px;
    border: 2px solid #00a388;
    border-radius: 15px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget_9" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>77</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>77</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_8" native="true">
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Workers</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_4">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Patients</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_8">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Products</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_5">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Prescription </string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_6">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Supplier</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_7">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Deliver</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>641</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#stackedWidget {
    background-color: white;
    width: 10px;
    border: 2px solid #00a388;
    border-radius: 15px;
}</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page_3">
    <widget class="QWidget" name="widget_10" native="true">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>47</y>
       <width>471</width>
       <height>309</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="4" column="0" colspan="2">
       <widget class="QWidget" name="widget_14" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLineEdit" name="lineEdit_10">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit{
	 color:#00a388;
	font: 700 12pt &quot;Segoe UI&quot;;
	
	;
}</string>
           </property>
           <property name="text">
            <string>pharmaceutical form</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="form">
           <property name="styleSheet">
            <string notr="true">/* Style for the ComboBox */
QComboBox {
    background-color: rgb(196, 255, 238);
    border: 2px solid #00a388;
    border-radius: 10px;
    padding: 2px;
    min-width: 120px;
}

</string>
           </property>
           <item>
            <property name="text">
             <string>Tablets</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Capsules</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Liquid Solutions</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Injectables </string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Topical Dosage Forms</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Inhaler   </string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Transdermal Patches</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QWidget" name="widget_15" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLineEdit" name="lineEdit_12">
           <property name="maximumSize">
            <size>
             <width>154</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit{
	 color:#00a388;
	font: 700 12pt &quot;Segoe UI&quot;;
	
	;
}</string>
           </property>
           <property name="text">
            <string>Expiration Date</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit_2">
           <property name="styleSheet">
            <string notr="true">#dateEdite{
background-color:rgb(0,0,0) ;
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QWidget" name="widget_11" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLineEdit" name="lineEdit_7">
           <property name="maximumSize">
            <size>
             <width>143</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit{
	 color:#00a388;
	font: 700 12pt &quot;Segoe UI&quot;;
	
	;
}</string>
           </property>
           <property name="text">
            <string>Product name</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameline_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>159</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#nameline_2 {
	background-color: rgb(196, 255, 238);
    border: 2px solid  #00a388;; /* Bordure rose */
    border-radius: 10px; /* Coins arrondis */
    padding: 2px;
}
</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QWidget" name="widget_12" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLineEdit" name="lineEdit_8">
           <property name="maximumSize">
            <size>
             <width>143</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit{
	 color:#00a388;
	font: 700 12pt &quot;Segoe UI&quot;;
	
	;
}</string>
           </property>
           <property name="text">
            <string>Unit price</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="priceLIne_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>159</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
	background-color: rgb(196, 255, 238);
    border: 2px solid  #00a388;; /* Bordure rose */
    border-radius: 10px; /* Coins arrondis */
    padding: 2px;
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QWidget" name="widget_13" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLineEdit" name="lineEdit_9">
           <property name="maximumSize">
            <size>
             <width>143</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit{
	 color:#00a388;
	font: 700 12pt &quot;Segoe UI&quot;;
	
	;
}</string>
           </property>
           <property name="text">
            <string>Quantity</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="qteliine_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>159</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
	background-color: rgb(196, 255, 238);
    border: 2px solid  #00a388;; /* Bordure rose */
    border-radius: 10px; /* Coins arrondis */
    padding: 2px;
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="Add_button_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>430</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#Add_button_2 {
	
	background-color: rgb(228, 0, 0);
    color: white; /* Texte blanc */
    border: none;
    border-radius: 10px; /* Coins arrondis */
    padding: 8px;
}</string>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>621</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#tableView{
background-color: rgb(196, 255, 238);
    border: 2px solid  #00a388;; /* Bordure rose */
    border-radius: 10px; /* Coins arrondis */
    padding: 2px;
}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="deleteline">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>380</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#deleteline{
background-color: rgb(196, 255, 238);
    border: 2px solid  #00a388;; /* Bordure rose */
    border-radius: 10px; /* Coins arrondis */
    padding: 2px;
}</string>
     </property>
     <property name="placeholderText">
      <string>Deleted id</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_button_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>410</y>
       <width>121</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#delete_button_2 {
	
	background-color: rgb(228, 0, 0);
    color: white; /* Texte blanc */
    border: none;
    border-radius: 10px; /* Coins arrondis */
    padding: 8px;
}</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame_2{
background-color: rgb(196, 255, 238);
    border: 2px solid  #00a388;; /* Bordure rose */
    border-radius: 10px; /* Coins arrondis */
    padding: 2px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../Downloads/Nouveau dossier/search.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_13">
        <property name="styleSheet">
         <string notr="true">#lineEdit_13{

background-color: rgb(196, 255, 238);
    border:none;; /* Bordure rose */
}</string>
        </property>
        <property name="placeholderText">
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QComboBox" name="tri">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* Style for the ComboBox */
QComboBox {
    background-color: rgb(196, 255, 238);
    border: 2px solid #00a388;
    border-radius: 10px;
    padding: 2px;
    min-width: 120px;
}

</string>
     </property>
     <property name="currentText">
      <string>Par defaut</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Par defaut</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>qte</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pdf">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>380</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pdf{
	background-color: rgb(228, 0, 0);
    color: white; /* Texte blanc */
    border: none;
    border-radius: 10px; /* Coins arrondis */
    padding: 8px;
}</string>
     </property>
     <property name="text">
      <string>Export pdf</string>
     </property>
    </widget>
    <widget class="QPushButton" name="excel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>440</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#excel{
	background-color: rgb(228, 0, 0);
    color: white; /* Texte blanc */
    border: none;
    border-radius: 10px; /* Coins arrondis */
    padding: 8px;
}</string>
     </property>
     <property name="text">
      <string>Export Excel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>601</width>
       <height>511</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="expiredProductsLayout"/>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
